chips/nrf52840/src/ieee802154_radio.rs

```diff
@@ -1298,24 +1349,24 @@ impl<'a> kernel::hil::radio::RadioData<'a> for Radio<'a> {
         buf: &'static mut [u8],
         frame_len: usize,
```

Comment:
I'm not entirely sure what the correct name is for this, but once again, `frame_len` seems somewhat misleading here since the frame does not contain the CRC here (that is to be added by hardware). There may not be a better name for this.